config/csm-versions.yaml

```diff
@@ -26,8 +26,19 @@ csm-metrics-powerstore : v1.11.0
 csm-metrics-powerscale : v1.8.0
 csm-metrics-powermax : v1.6.0
 karavi-topology : v1.11.0

```

Comment:
Observability and replication tags are not aligned to 1.14 release like other image versions.

Reply:
nice catch, let me fix them and verify others.